rules/run_binary.bzl

```diff
@@ -44,7 +44,8 @@ def _impl(ctx):
     }
     ctx.actions.run(
         outputs = ctx.outputs.outs,
-        inputs = depset(direct = ctx.files.srcs, transitive = [tool_inputs]),
+        inputs = depset(direct = ctx.files.srcs),
```

Comment:
Can `inputs` just be ctx.files.srcs or does it have to be wrapped in a depset?

Reply:
Thanks, the depset was indeed useless